api/filters/authn/oauth/authenticator.go

```diff
@@ -96,13 +96,13 @@ func newOIDCConfig(options *options) *goidc.Config {
 // Authenticate returns information about the user by obtaining it from the bearer token, or an error if security is unsuccessful
 func (a *oauthAuthenticator) Authenticate(request *http.Request) (*web.User, security.Decision, error) {
 	authorizationHeader := request.Header.Get("Authorization")
-	if authorizationHeader == "" || !strings.HasPrefix(strings.ToLower(authorizationHeader), "bearer") {
+	if authorizationHeader == "" || !strings.HasPrefix(strings.ToLower(authorizationHeader), "bearer ") {
 		return nil, security.Abstain, nil
 	}
 	if a.Verifier == nil {
 		return nil, security.Abstain, errors.New("authenticator is not configured")
 	}
-	token := strings.TrimPrefix(authorizationHeader, "Bearer ")
+	token := strings.TrimSpace(authorizationHeader[len("Bearer "):])
```

Comment:
not sure about this one, but as far as i remember the authentication token value (oauth2 spec) should be ignorecased. Meaning both Bearer and bearer should be expected here.

Reply:
It doesn't matter what the case is. I use the length of the "Bearer " string.

Reply:
Yeah with this change you added len, my bad - for some reason i was looking at the trimpredix version